packages/block-library/src/list/block.json

```diff
@@ -29,6 +26,7 @@
 	},
 	"supports": {
 		"className": false,
+		"__experimentalColor": true,
```

Comment:
What about "gradients" do we want to enable them?

Reply:
I figured we should get regular background colors working first, but I don't see any reason to exclude gradients.